example/main.go

```diff
@@ -22,14 +26,17 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
+	// [developer-docs.sdk.go.client-initialization]-end
 
 	item := createAndGetItem(client)
 	getAndUpdateItem(client, item.VaultID, item.ID)
-	listVaultsAndItems(client)
+	listVaultsAndItems(client, item.VaultID)
 	resolveSecretReference(client, item.VaultID, item.ID, "username")
+	deleteItem(client, item.VaultID, item.ID)
 }
 
-func listVaultsAndItems(client *onepassword.Client) {
+func listVaultsAndItems(client *onepassword.Client, vaultID string) {
+	// [developer-docs.sdk.go.list-vaults]-start
```

Comment:
Do we not want to include the function signature? and this goes for all.

Reply:
No, this is merely an implementation detail on our side. We should aim to only exhibit what's relevant for the user, and in this block, that is how to list items and vaults. Whether they get the client and vault ID as parameters in a function or instantiate them, it's irrelevant for the overall functionality.